tests/Unit/TagTest.php

```diff
@@ -26,11 +26,11 @@ public function test_it_belongs_to_many_contacts()
         $account = factory('App\Account')->create([]);
         $contact = factory('App\Contact')->create(['account_id' => $account->id]);
         $tag = factory('App\Tag')->create(['account_id' => $account->id]);
-        $contact->tags()->sync($tag->id);
+        $contact->tags()->sync([$tag->id => ['account_id' => $account->id]]);
```

Comment:
No you don't need that to test what the tests do.

Reply:
Yes it's necessary, for the same reason as listed above (the debt example).

Reply:
Thanks!